tests/tests.js

```diff
@@ -617,7 +617,7 @@ describe('advanced tests', function() {
 			jsesc(testArray, {
 				'json': false
 			}),
-			'[undefined,Infinity,Infinity,-Infinity,-Infinity,0,0,0,0,0,0,function anonymous(\n) {\n\n},\'str\',function zomg() { return \'desu\'; },null,true,true,false,false,{\'foo\':42,\'hah\':[1,2,3,{\'foo\':42}]}]',
+			'[undefined,Infinity,Infinity,-Infinity,-Infinity,0,0,0,0,0,0,function anonymous() {\n\n},\'str\',function zomg() { return \'desu\'; },null,true,true,false,false,{\'foo\':42,\'hah\':[1,2,3,{\'foo\':42}]}]',
```

Comment:
This breaks the test on modern Node.js due to https://v8.dev/features/function-tostring. Will undo